ratis-docs/content/cli.md

```diff
@@ -23,6 +23,37 @@ Ratis-shell is the command line interface of Ratis.
 > Ratis-shell is currently only **experimental**.
 > The compatibility story is not considered for the time being.
 
+
+## Get the ratis-shell source 
+
+>Mention: Ratis shell is available starting from Apache Ratis 2.3.0.
+
+### Build `ratis-shell` from src tarball
+Get the ratis src tarball from [Ratis Source Download](https://ratis.apache.org/downloads.html)
+
+```
+$ tar -C <DST_DIR> -zxvf apache-ratis-*-src.tar.gz
+
+$ cd <DST_DIR>/apache-ratis-*-src
+
+$ mvn -DskipTests -Prelease -Papache-release clean package assembly:single
+...
+[INFO] BUILD SUCCESS
+
+$ tar -C <DST_DIR>/ratis-shell -xzf ratis-assembly/target/apache-ratis-2.3.0-SNAPSHOT-src.tar.gz
```

Comment:
It should use the shell.tar.gz but not src.tar.gz.
```
$ tar -C <DST_DIR>/ratis-shell -xzf ratis-assembly/target/apache-ratis-<VERSION>-shell.tar.gz
```

Reply:
done